packages/terra-clinical-result-view/src/ResultView.scss

```diff
@@ -35,25 +35,81 @@
   }
 
   .result-view-indicator {
-    padding-right: var(--result-view-indicator-padding-right, 0.1875rem);
+    padding-right: var(--result-view-indicator-padding-right, 0.1875em);
+
+    &.critical {
+      color: var(--result-view-text-color, #c00);
+    }
+
+    &.low {
+      color: var(--result-view-low-color, #0053e6);
```

Comment:
These should probably be more specific to what they are themeing as well as name-spaced to the terra component to avoid potential name collisions : `--terra-clinical-result-view-normalcy-indicator-low`

Reply:
^ It actually looks like this themable naming is reused in different places with fallback set a few times..